### DISCUSSION TITLE
Found two 'bug's in the Sequence Play mode

### DISCUSSION THREAD

**Discussion by listenForward** [bangcorrupt/hacktribe#331] [General]
Hey there BangCorrupt and HackTribe community !

I am still very much enjoying the HackTribe in allowing me to mix all the Synth's Oscillators and UserSamples (especially now that I got my own sample-set organized using [Sample Manager from Ingolf Koch](http://appification.de/ElectribeSampleManager/)

However, now that I'm getting back into the flow of playing the E2 my way, that involves using the pads for Seq Play, or Seq Play 1st instead of Normal mode.

When I first leaned about the HackTribe firmware, I brainstormed and wish-list-ed a whole lot of ideas to potentially expand the play-ability of Seq Play pad work in a discussion/post [HERE](https://github.com/bangcorrupt/hacktribe/discussions/203).
...but now that I've actually started playing the HackTribe my way, I've found two things that may be 'bugs'.
I don't think these things worked/bugged this way in the normal 'Tribe firmware, and I'm not yet ready to switch my (lone) E2 back and forth for testing...
so I'll describe them and see if BangCorrupt or others can sanity-check.

1. If a track's Sequence has triggers on the first 2 steps/triggers (16th notes)... if the Pad Mode is changed to SeqPlay1st, then the pad will the 1st trigger/step when pressed DOWN, then play the 2nd trigger/step when the pad is released UP.
2. If I try to use pad for SeqPlay on tracks 13/14 and/or 15/16 that are set to choke-pairs, I find that the Left/odd of the pair drops it's starting Step#1 IF/when they both compete for Step#1... whether or not  if/when they are of different bar/step-lengths and are doing any poly-rhythmic rotation/re-alignment... There is still drop-out of the Step/Trig#1.

I do not recall experiencing these symptoms on a Stock E2.
Can anyone else confirm if/how this is a Stock and/or HackTribe bug ?

cheers
ListenForward.

**Comment by bangcorrupt** [comment]
Hi, if you can share some pattern files that demonstrate these issues I will try to reproduce them.  I have had a bug where the pad triggers notes when pressed and released, it could be related.
